### ISSUE TITLE
Fix describe_table to work with views

### ISSUE THREAD

**Issue by kadrlica** [mgckind/easyaccess#46]
As best I can figure, this is very rarely a problem with objects in DESSCI because almost all VIEWs are also SYNONYMs. The problem does arise for VIEWs like 'WDF' and 'WGB' in DESOPER. However, if you prefix these VIEWs by their owner, i.e., PROD.WDF or PROD.WGB then `describe_table` works well.

I think the issue is in `get_tablename_tuple` where the order of operations is:
1. Look for a table in `ALL_TAB_COLUMNS` where the `OWNER = %(schema)s`
2. Look for a user synonym in `USER_SYNONYMS` named `TABLE`
3. Look for a public synonym `ALL_SYNONYMS` with `OWNER = 'PUBLIC`

This noticeably omits PUBLIC objects where no schema is provided. In general, these objects that lack a schema are being picked up by the third criteria which checks for public synonyms. I _think_ that it should be possible to remedy the issue with views by adding another option which searches for the table name without an owner if the schema is omitted from the query.

**Comment by kadrlica** [comment]
A possibly related problem is that I can't tab complete for the `Y1A1_COADD_COPPER` table. Maybe it's tab completion that is broken for views?

**Comment by mgckind** [comment]
it is not related, views are not included in the metadata table which is used for the autocompletion. although I can autocomplete `Y1A1_COADD_COPPER` and its actually a table

**Comment by kadrlica** [comment]
So I guess the question is then why didn't it make it into my metadata? I've noticed a couple instances (though I don't remember what exactly yet). Probably a topic for a new issue.

**Comment by mgckind** [comment]
Sometimes, when loading the metadata is taking too long (more than 1 min), it will skip that  step, but in that case neither of the tables will work with autocompletion

**Comment by kadrlica** [comment]
I'm not sure if this is related, but the following query is not working for me:

```
DESDB ~> describe_table Y1A1_GOLD_1_0_2_D04
Table not found.
```

However, the table exists (and is recognized as a table):

```
DESDB ~> select object_name from all_objects where object_name like 'Y1A1_GOLD%' order by object_name;
10 rows in 0.21 seconds

                  OBJECT_NAME
1                   Y1A1_GOLD
2             Y1A1_GOLD_1_0_0
3             Y1A1_GOLD_1_0_1
4   Y1A1_GOLD_1_0_1_D04_NOSNS
5         Y1A1_GOLD_1_0_1_D10
6       Y1A1_GOLD_1_0_1_DFULL
7             Y1A1_GOLD_1_0_2
8         Y1A1_GOLD_1_0_2_D04
9         Y1A1_GOLD_1_0_2_D10
10      Y1A1_GOLD_1_0_2_DFULL
```

**Comment by mgckind** [comment]
describe_table nsevilla.Y1A1_GOLD_1_0_2_D04 works though.

the describe_table doesn't handle owner, same with find_table

**Comment by mgckind** [comment]
In another words, there is an implicit synonym for DES_ADMIN tables, all the rest need the owner in front. There is also user_table to list tables by specific user.  maybe is find_table that needs to be fixed. 
But this issue is related to views and those Y1A1_ are all tables as I just checked.

**Comment by kadrlica** [comment]
Oh, good point, for some reason I thought we had actually gotten these tables promoted to DES_ADMIN (I'm remembering a conversation that we had in February).

**Comment by mgckind** [comment]
This is also fixed in #113  and other changes in metadata.
